caseconsulting/vue-expense-app
# POR 2641 - migrate /shared folder to composition api

Ticket Link [POR-2641](https://consultwithcase.atlassian.net/browse/POR-2641)

Migrated components/shared folder to compositions api

Files changed/What to test
1. AnniversaryCard.vue, used in profile page, home, and my budgets page 
    - Make sure you can change between the budget years
2. AvailableBudgets.vue,  used in profile and home page
3. AvailableBudgetsSummary.vue, should show info on selected budget
4. ContactEmployeesModel.vue, used in reports page
    - Make sure adding employees works and mail button works
5. HighFivesTable.vue, lol
6. PTOCashOutForm.vue, used in PTOCashoutsTable and in timesheets widget when using the cash out button
7. PTOCashOutsTable.vue, used in PTOCashouts and reinbursements
    - Make sure approving, creating, and editing pto works
8. UserAvatar.vue



[POR-2641]: https://consultwithcase.atlassian.net/browse/POR-2641?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
